docs/index.rst

```diff
@@ -2,42 +2,74 @@
 sunpy core Documentation
 ************************
 
-The sunpy core package is a community-developed, free and open-source solar
-data analysis environment for Python.
+``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
+
+
+New Users
+==========
+
+For help installing ``sunpy`` refer to our :ref:`installation guide <installing>`.
+
+
+.. todo::
+
+    if #6615 is completed link here
+    Introduction to sunpy core
+
+If you are a first time user or new to Python, our **tutorial** provides a walkthrough on how to use the key features of sunpy core.
+Once you've installed sunpy, start here.
+
+* :ref:`guide`
```

Comment:
I thought you had put a toctree here?

Reply:
see my comment in main thread, because of #6616 it now breaks the toctree here